jdi-light-html-tests/src/test/java/io/github/epam/test/data/ListData.java

```diff
@@ -0,0 +1,31 @@
+package io.github.epam.test.data;
+
+import io.github.com.custom.Result;
+
+public class ListData {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ListData' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)